ui/src/services/NodeVolumesUtils.js

```diff
@@ -267,6 +267,7 @@ export const getVolumeListData = createSelector(
             ? Math.round(volumeCurrentLatency?.value[1]) + ' µs'
             : undefined,
         errorReason: volume?.status?.conditions[0]?.reason,
+        namespace: volumePVC?.metadata?.namespace,
```

Comment:
Actually the NodeVolumeUtils should return the volume list only used by the Volume table. 

So I would rather not add a namespace field here, but get the **namespace** from the **PVC** in the `VolumePageContent.js`and pass it to `<VolumeMetricsTab/>`

The reason is that to have an isolated function to retrieve the data for VolumeTable. It will easier for us to do the test.